ecosystem/platform/server/app/assets/stylesheets/typekit.css

```diff
@@ -0,0 +1,105 @@
+/*
+ * The Typekit service used to deliver this font or fonts for use on websites
+ * is provided by Adobe and is subject to these Terms of Use
+ * http://www.adobe.com/products/eulas/tou_typekit. For font license
+ * information, see the list below.
+ *
+ * apparat:
+ *   - http://typekit.com/eulas/00000000000000007735ed00
+ *   - http://typekit.com/eulas/00000000000000007735ed01
+ *   - http://typekit.com/eulas/00000000000000007735ed12
+ *   - http://typekit.com/eulas/00000000000000007735ed14
+ * apparat-semicond:
+ *   - http://typekit.com/eulas/00000000000000007735ed2b
+ *   - http://typekit.com/eulas/00000000000000007735ed2e
+ *   - http://typekit.com/eulas/00000000000000007735ed32
+ *   - http://typekit.com/eulas/00000000000000007735ed39
+ *   - http://typekit.com/eulas/00000000000000007735ed3b
+ * apparat-semicond-light:
+ *   - http://typekit.com/eulas/00000000000000007735ed38
+ * lft-etica-mono:
+ *   - http://typekit.com/eulas/00000000000000007735c995
+ *   - http://typekit.com/eulas/00000000000000007735c99b
+ *
+ * © 2009-2022 Adobe Systems Incorporated. All Rights Reserved.
+ */
+/*{"last_published":"2022-05-12 19:42:21 UTC"}*/
+
+@import url("https://p.typekit.net/p.css?s=1&k=ifl8enc&ht=tk&f=44164.44166.47749.47752.47759.47761.47789.47792.47794.47796.47800.47802&a=109597199&app=typekit&e=css");
+
+@font-face {
+font-family:"lft-etica-mono";
```

Comment:
Are all of these guaranteed to be static URLs? Ie is doing this something they are okay with?  I want to make sure we don’t run afoul of licensing or their upgrades that change the JS but not anything else

Reply:
They don't allow you to host the font files, but I don't see any reason why you can't host the CSS pointing to the font files.

https://helpx.adobe.com/fonts/using/font-licensing.html#web-host

What we could do if we're worried about it is update the typekit config to be font-display: optional (instead of auto), and link in https://use.typekit.net/ifl8enc.css to load the fonts. This will slow down initial load a bit since it's a different origin (i.e. TLS handshake latency).

Reply:
Updated to load ifl8enc.css.